PREVERSION.md

```diff
@@ -28,57 +27,90 @@
 ## enhancements
 - [x] squash lite supports multiple connections (use localport flag)
 ## cli
-- [ ] distinguish between lite/agent mode
-- [ ] use cobra
-- [ ] auto gen docs
+- [x] distinguish between lite/agent mode
+- [x] use cobra
+- [x] auto gen docs
 ## tutorial
-- [ ] install sample apps from command line
+- [x] install sample apps from command line
 ## backlog
-- [ ] allow in/out-of cluster to be configured as a flag
-- [ ] allow installation of squash agent from cli
+- [x] allow in/out-of cluster to be configured as a flag - will not do
+- [x] allow installation of squash agent from cli
+## outstanding
+- [ ] (P0) interactive input for RBAC mode
+- [ ] (P0) decide what to call RBAC/Agent
+- [ ] (P0) choose container in interactive mode
+- [ ] (P1, testing, docs) python support updates
+- [ ] (P1, testing, docs) java support updates
+- [ ] (P1, testing, docs) nodejs support updates
+- [ ] (P1, future) simplify DebugAttachment.State options
+## e2e tests
+- [x] rbac mode
+- [ ] (P1) non rbac mode
 
 ## Release tasks
-- update the docs
-  - no server required
-  - new cli flags
-  - caveats for each debugger/ide
-- tag a release update
-- push a new image to dockerhub
-- update the tag in `github.com/solo-io/squash/contrib/kubernetes/squash-client.yml`
+- [ ] (P0) update the docs
+  - [ ] (P0) no server required
+  - [x] new cli flags
+  - [ ] (P1) caveats for each debugger/ide
+- [ ] (P0) (small) tag a release update
+- [ ] (P0) (small) push a new image to dockerhub
+- [ ] (P0) (small) update the tag in `github.com/solo-io/squash/contrib/kubernetes/squash-client.yml`
```

Comment:
some more release tasks:
there are a few kubesquash commits that were contributed recently that we should port.

also port the CI process that releases everything automatically.
we probably want to merge the extensions (i.e. port the download an version pinning from the kube squash extension)

Reply:
added to the list